src/lib/y2storage/proposal/autoinst_devices_planner.rb

```diff
@@ -89,14 +89,65 @@ def planned_devices(drives_map)
       # @return [AutoinstIssues::List] List of AutoYaST issues to register them
       attr_reader :issues_list
 
+      # Returns an array of planned partitions for a given disk or the disk
+      # itself if there are no partitions planned
+      #
+      # @param disk [Disk,Dasd] Disk to place the partitions on
+      # @param drive [AutoinstProfile::DriveSection] drive section describing
+      #   the layout for the disk
+      # @return [Array<Planned::Partition, Planned::StrayBlkDevice>] List of planned partitions or disks
+      def planned_for_disk(disk, drive)
+        result = []
+
+        # if there are no partitions use the entire device
+        if drive.partitions.empty?
+          result = planned_for_full_disk(drive)
```

Comment:
This does the same in the case in which we have just one partition with `partition_nr=0, create=false` (which was the scope of the PBI) and if we have an empty list of partitions (which is a different scenario that is reflected in the current AutoYaST documentation).

Reply:
The created layout is the same as before in this case.